run

```diff
@@ -2,6 +2,18 @@
 set -x
 set -eo pipefail
 
+###
+# Check "ODO_S2I_DEPLOYMENT_DIR" environment variable and if it's present,
+# copy content from /opt/app-root/backup directory to "ODO_S2I_DEPLOYMENT_DIR"
+# Ref: https://github.com/redhat-developer/odo/issues/445
+if [ -z "$ODO_S2I_DEPLOYMENT_DIR" ]
+then
+   echo "\$ODO_S2I_DEPLOYMENT_DIR is empty"
```

Comment:
I don't think that user will need to see this message. It won't tell him/her anything useful.